xsd/gml-bison.xsd

```diff
@@ -51,10 +48,8 @@
 				<!--	<element name="interior" type="gml:LinearRingType" minOccurs="0"/>	-->
 			</sequence>
 			<attribute ref="gml:id" use="required"/>
-			<!-- default coördinatensysteem gebruiken 
-				<attribute name="srsName" type="anyURI"/>
-				<attribute name="srsDimension" type="positiveInteger"/>
-				-->
+			<attribute name="srsName" type="anyURI" use="optional" fixed="EPSG:28992" />
```

Comment:
De twee srsXxxx attributen zitten al op ('posList' in 'LinearRing' in) het inliggende element 'exterior', dus hoeven niet op Polygon niveau herhaald te worden.